ci/run_test/run_test_post_always_node.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -x
+
+cd $DAOS_BASE
+mkdir run_test.sh
+mkdir vm_test
+if ls /tmp/daos*.log > /dev/null; then
+  mv /tmp/daos*.log run_test.sh/
+fi
+if ls /tmp/dnt*.log > /dev/null; then
+  mv /tmp/dnt*.log vm_test/
+fi
+# servers can sometimes take a while to stop when the test is done
+x=0
+while [ \"\\\$x\" -lt \"10\" ] &&
```

Comment:
(lint) -lt is for integer comparisons. Use \< instead. [SC2130]
(lint) This expression is constant. Did you forget the $ on a variable? [SC2050]